### DISCUSSION TITLE
Make a relation from one side

### DISCUSSION THREAD

**Discussion by BishoyNarouz** [prisma/prisma#6017] [Q&A] [answered]
is this valid to make (one to one or one to many) from one side only?
example
```prisma
model Category {
  id   String  @id @default(uuid())
  name String
}

model Product {
  id               String  @id @default(uuid())
  name         String
   category   Category    @relation(fields: [catId], references: [id])
    catId         String
}
```
is this valid? or i must add  this line in category model (products  Product[] )

**Comment by ryands17** [comment, marked as answer]
@BishoyNarouz 👋 

The relation will be added in the database on one side only. The field created on the other side is just a virtual field to make it easy to fetch the relation from both sides. So you must add this. Even if you don't, it will automatically be added by the Prisma plugin in VSCode on save.

You can read more about relations [here](https://www.prisma.io/docs/concepts/components/prisma-schema/relations).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
